### ISSUE TITLE
Separating ranked leagues & adding perks to the leaderboards

### ISSUE THREAD

**Issue by peth-yursick** [MetaFam/TheGame#1703]
### What would you like to be added?

Separating the categories on leaderboards & adding perks.

Like categories are separated in the Academy page, I think ranked leagues should be separated out by subheadings.
There should be a little question mark that a user can hover, to see the perks of that specific league.
Mockup:
![image](https://github.com/MetaFam/TheGame/assets/55857222/6f0dd083-7cf9-45da-a188-2e808b2cd3b6)

All perks (need to be updated):
![image](https://github.com/MetaFam/TheGame/assets/55857222/8c52f93f-ce07-4620-bf0f-756c8a39ba55)

After its done, pls tag @peth-yursick so he can add the link everywhere its missing.

### Why is this needed?

We've had these perks for years but removed them from the new signup pages so they're less cluttered.
It also makes more sense to have them inside MetaGame rather than at the gates (landing page).
I think it would make most sense for them to be on the leaderboard, so they can be seen by more users.

They are referred to from the signup pages, from the purplepaper, the Seeds page & the patron's path - but are nowhere to be found currently.

**Comment by dysbulic** [comment]
To reiterate a criticism I've voiced in the past, who cares about MetaGame's leaderboard other than MetaGamers? Wouldn't a better use of developer time be making the leaderboards customizable for different guilds?

**Comment by peth-yursick** [comment]
> who cares about MetaGame's leaderboard other than MetaGamers

Currently possibly not even MetaGamers 🤣
Having clearer separation & perks would hopefully improve that, which would be proven in the next step when I try to reactivate some of the old players at the end of the month/early feb.

If we got MetaGamers to care more about the leaderboard, we would also prove to others that a leaderboard is a useful product, which would then make sense to customize for other guilds. So far I've only spoken to a few guilds but none of them seemed very interested..

Tl;dr I agree but still think this is a necessary change. Separation should be trivial & perks already exist, so really should be displayed somewhere if we want to use them to incentivize more people to climb up the leaderboard 🤷‍♂️
